website/docs/prelinkcommands.md

```diff
@@ -1,4 +1,4 @@
-Specifies shell commands to run after the source files have been compiled, but before the link step.
+Specifies shell commands to run after the source files have been compiled, but before the link step (when supported, else behave as [prebuildcommands](prebuildcommands.md)).
```

Comment:
This is an awkward way of phrasing this. Something like this would work better:
```suggestion
Specifies shell commands to run after the source files have been compiled, but before the link step (if unsupported by the action, it will be treated the same as [prebuildcommands](prebuildcommands.md)).
```

Reply:
Applied.